bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/ladda/LaddaAjaxLink.java

```diff
@@ -78,23 +78,11 @@ protected void onInitialize() {
      * @param color The color for the spinner
      * @return {@code this}, for chaining
      */
-    public LaddaAjaxLink<T> setSpinnerColor(String color) {
+    public LaddaAjaxLink<T> setSpinnerColor(LaddaBehavior.Color color) {
```

Comment:
Is this extensible? I.e is it possible to have custom colors not present in enum? If so, better to keep String setter too

Reply:
I guess this should be limited to this `https://getbootstrap.com/docs/4.4/utilities/colors/#color` list
Additional parameter like `customCssClass` can be added